### ISSUE TITLE
Minor accessibility enhancement: Tab between value box

### ISSUE THREAD

**Issue by gorjious** [Mark2Mark/variable-font-preview#16]
I appreciate the ability for changing values in the number boxes using up/down arrows (also the ability to increase by 10 with shift). 

In tandem with this input method via the keyboard, it would be handy to add be able to Tab between the parameter values.

**Comment by Mark2Mark** [comment]
(… and by 100 with shift+cmd) :)

Good call! This was in my head but never ticketed, thanks for opening it for me here!

**Comment by gorjious** [comment]
What about in the opposite direction? 
+/- .1 units  

Not sure if it will conflict with something else but maybe using `option + up/down arrow`?

I suggested something similar for FontGoggles because it doesn't have any of the keyboard accessibility features.
https://github.com/justvanrossum/fontgoggles/issues/113#issuecomment-677791047
